test/activities/quizzes/QuizEntity.js

```diff
@@ -125,6 +126,20 @@ describe('QuizEntity', () => {
 		});
 	});
 
+	describe('passwords', () => {
+		describe('canEditPassword', () => {
+			it('returns true when password is editable', () => {
+				var quizEntity = new QuizEntity(editableEntity);
+				expect(quizEntity.canEditPassword()).to.be.true;
+			});
+
+			it('returns false when password is not editable', () => {
+				var quizEntity = new QuizEntity(nonEditableEntity);
+				expect(quizEntity.canEditPassword()).to.be.false;
+			});
```

Comment:
probably wanna add tests like I have on line 115 to line 124 that your `password()` function reads the password from the entity correctly.